Documentation/contributing/development/dev_setup.rst

```diff
@@ -450,7 +450,9 @@ Minor version
 
 #. Run ``git add vendor/ test/provision/manifest/ Documentation/ && git commit -sam "Update k8s tests and libraries to v1.28.0-rc.0"``
```

Comment:
Please also add that we need to bump the k8s version on these files:

- .github/actions/ginkgo/main-prs.yaml
- .github/actions/ginkgo/main-scheduled.yaml
- .github/actions/set-env-variables/action.yml
- contrib/scripts/devcontainer-setup.sh
- .github/actions/ginkgo/main-focus.yaml

(Similar to what was done in https://github.com/cilium/cilium/pull/34778 and in https://github.com/cilium/cilium/pull/34773)